python/cudf/dataframe/series.py

```diff
@@ -1270,6 +1270,34 @@ def to_hdf(self, path_or_buf, key, *args, **kwargs):
         import cudf.io.hdf as hdf
         hdf.to_hdf(path_or_buf, key, self, *args, **kwargs)
 
+    def rename(self, index=None, copy=True):
+        """
+        Alter Series name.
+
+        Change Series.name with a scalar value.
+
+        Parameters
+        ----------
+        index : Scalar, optional
+            Scalar to alter the Series.name attribute
+        copy : boolean, default True
+            Also copy underlying data
+
+        Returns
+        -------
+        Series
+
+        Difference from pandas:
+          * Supports scalar values only for changing name attribute
+          * Not supporting: inplace, level
+        """
+        out = self.copy(deep=False)
+        out = out.set_index(self.index)
+        if index:
+            out.name = index
+
+        return out.copy(deep=copy)
```

Comment:
You're not supporting the advanced `Series.rename` functionality that uses a mapping or function to change labels. Your call.
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.rename.html

Reply:
For series that applies it against the Index which we in general don't support across the board yet. Same reason we don't support axis for DataFrame.